### DISCUSSION TITLE
Two snap rules - which is better

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#91] [Q&A]
1. This is the first method -  `Can save multiple snaps from  the same webpage.`

![image](https://user-images.githubusercontent.com/67494083/158694330-d31d2cd7-1f58-48ff-a829-f7edf782c8b8.png)

2. This the second method - `only saves the first snap from a website`

![image](https://user-images.githubusercontent.com/67494083/158694466-0b402d92-ce0b-485c-9d1a-fab4e8dcce7f.png)

![image](https://user-images.githubusercontent.com/67494083/158694640-2b8baccc-9648-410f-8f52-4aedcd649aef.png)

Why is the second one not able to save any snaps after the first snap from a webpage.

**Comment by buggyj** [comment]
the rules in the column 'field initials' are only applied one time when a new tiddly is created. 'field modifiers' apply rules each time the clip is executed.

Note that the mode 'append' means that when the clip is applied more than one, the content created by the rule in the 'body' column will be appended to the existing content of the text of the tiddler. There is also the mode 'prepend' that put the new content before the existing content.

**Reply by arunnbabu81** [reply]
> the rules in the column 'field initials' are only applied one time when a new tiddly is created. 'field modifiers' apply rules each time the clip is executed.
> 
> Note that the mode 'append' means that when the clip is applied more than one, the content created by the rule in the 'body' column will be appended to the existing content of the text of the tiddler. There is also the mode 'prepend' that put the new content before the existing content.

Thank you. With your help, I was able to correct the error.

**Reply by arunnbabu81** [reply]
Which method do you recommend. I find the second method much simpler and avoids creation of an extra image tiddler.

**Reply by buggyj** [reply]
I don't have a preference. I don't like to have lots of tiddlers, but if you wanted to move the image files to another disk location (at future time) it would be much easier to do so with separate image tiddlers.
